pkg/crc/segment/segment.go

```diff
@@ -92,6 +92,18 @@ func (c *Client) Upload(ctx context.Context, action string, duration time.Durati
 	})
 }
 
+func addConfigTraits(c *crcConfig.Config, in analytics.Traits) analytics.Traits {
+	return in.
+		Set("proxy", isProxyUsed(c)).
+		Set(config.NetworkMode, c.Get(config.NetworkMode).AsString()).
+		Set(config.EnableClusterMonitoring, c.Get(config.EnableClusterMonitoring).AsBool()).
+		Set(config.ExperimentalFeatures, c.Get(config.ExperimentalFeatures).AsBool())
+}
+
+func isProxyUsed(c *crcConfig.Config) bool {
+	return c.Get(config.HTTPProxy).AsString() != "" || c.Get(config.HTTPSProxy).AsString() != ""
```

Comment:
I forgot if when `http_proxy` is set in the environment, it's reflected in the config?

Reply:
I would say no. Viper uses CRC_HTTP_PROXY.

Reply:
Imo it's very misleading to have false negative in telemetry regarding proxy usage. If `http_proxy` is set in the env, telemetry will report "proxy: false" while the cluster will be using a proxy.